examples/coexist_klog_v1_using_logr/coexist_klog_v1.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"flag"
+	"github.com/go-logr/logr"
+
+	"k8s.io/klog"
+	klogv2 "k8s.io/klog/v2"
+)
+
+type klogLogger struct{}
+
+func (kw klogLogger) Info(msg string, keysAndValues ...interface{}) {
+	klog.Info(msg, keysAndValues)
+}
+
+func (kw klogLogger) Enabled() bool {
+	return true
+}
+
+func (kw klogLogger) Error(err error, msg string, keysAndValues ...interface{}) {
+	klog.Error(err, msg, keysAndValues)
```

Comment:
same question about line numbers, vmodule, and klog2.Fatalf behavior

Reply:
[dims@dims-a02 18:21] ~/go/src/k8s.io/klog/examples/coexist_klog_v1_using_logr ⟩ go run coexist_klog_v1.go && cat -p myfile.log && rm myfile.log
Log file created at: 2020/05/08 18:21:09
Running on machine: dims-a02
Binary: Built with gc go1.13.10 for darwin/amd64
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
I0508 18:21:09.815897   31798 coexist_klog_v1.go:47] hello from klog v1!
I0508 18:21:09.816189   31798 coexist_klog_v1.go:48] hello from klog v2!
I0508 18:21:09.816196   31798 coexist_klog_v1.go:49] beware from klog v2!
E0508 18:21:09.816202   31798 coexist_klog_v1.go:50] error from klog v2!
I0508 18:21:09.816205   31798 coexist_klog_v1.go:51] nice to meet you